cpp/player.h

```diff
@@ -14,8 +14,11 @@ class Unit;
 
 class Player {
 public:
-	Player(unsigned int number);
+	Player(unsigned int number, std::string name);
 
+	/**
+	 * numbers 0-7
+	 */
 	const unsigned int player_number;
```

Comment:
the comment was meant to imply a range 0..n-1 rather than the colour value which is 1..n